forte_wrapper/elastic/elastic_search_processor.py

```diff
@@ -48,7 +48,8 @@ def default_configs(cls) -> Dict[str, Any]:
             "query_pack_name": "query",
             "index_config": ElasticSearchIndexer.default_configs(),
             "field": "content",
-            "response_pack_name_prefix": "passage"
+            "response_pack_name_prefix": "passage",
+            "only_pass_content": True
```

Comment:
It might be hard for users to understand what `only_pass_content` mean, can you add to the docstring, explain what it achieves?